### DISCUSSION TITLE
Use Pixijs as the engine

### DISCUSSION THREAD

**Discussion by fwx5618177** [ton-org/game-engines-sdk#9] [Q&A] [answered]
Maybe we could use @ton/phaser-sdk and pixijs? Use Pixjs draw more complex scenes, and SDK to set the connect wallet button?

**Comment by barinbritva** [comment, marked as answer]
Hi @fwx5618177,

Not sure I've got your question right. Am I right you want to use ton SDK in your Pixi.js project?
If so, you could give a try to `vanilla-sdk` version. There is no render connect button wallet, but it has all the other features.
`npm install --save @ton/vanilla-sdk@beta`

Did you mean something else?

**Reply by fwx5618177** [reply]
Got it. Thanks!
